extras/DevelopersREADME.md

```diff
@@ -0,0 +1,66 @@
+DQD Developers README
+====================
+
+Dev Setup 
+====================
+1. R setup: https://ohdsi.github.io/Hades/rSetup.html  
+2. Local OMOP CDM setup 
+
+   If you already have a CDM available for development work/testing, you may skip this step 
+   
+   a. Install Postgres and create a localhost server 
+   
+   b. Create a new database in localhost for your test CDM, and create a schema in that database for the CDM tables 
+   
+   c. Using the [CDMConnector](https://cran.r-project.org/web/packages/CDMConnector/index.html)  package:
+     
+      <code>
+             i.  Download a sample OMOP CDM into a DuckDB database, as documented here 
+      
+     ii. Copy the CDM into your local Postgres database, as documented here
+</code>
```

Comment:
<img width="998" alt="Screenshot 2023-10-02 at 11 29 52 PM" src="https://github.com/OHDSI/DataQualityDashboard/assets/38965545/587752b9-afda-4529-9645-7fcac2402e86">

the sub-bullets are still messed up - see above :)

Reply:
thanks for the change!  one final addition will be to add the hyperlinks to CDMConnector to the 2 "here"s like in the word doc.  then should be good to go :)